webpack/JobWizard/steps/HostsAndInputs/HostPreviewModal.js

```diff
@@ -22,16 +22,16 @@ export const HostPreviewModal = ({ isOpen, setIsOpen, searchQuery }) => {
     >
       <List isPlain>
         {hosts.map(host => (
-          <ListItem key={host}>
+          <ListItem key={host.name}>
             <Button
               component="a"
-              href={foremanUrl(`/hosts/${host}`)}
+              href={foremanUrl(`/new/hosts/${host.name}`)}
```

Comment:
Shouldn't it be dependent on a switch?
@ron-lavi?

Reply:
You are right, we should check user's settings here. I'll remove it and open a separate PR for that.

Reply:
I saw the same behaviour across the app so I think it is OK..

Probably we can try to load the correct host path to the React Context Settings and use that helper across the app ( but that's for another PR ). :)